pkgs/development/tools/zigmod/tests/clap/default.nix

```diff
@@ -0,0 +1,18 @@
+{ fetchurl, buildZigmodPackage }:
+
+buildZigmodPackage {
+  pname = "zigmod-clap-test";
+  version = "1";
+
+  src = ./.;
+
+  lockFile = fetchurl {
+    url = "file:///" + ./zigmod.lock;
```

Comment:
No, it might be any URL. We separate updates of the repository and the lock file now, so the sources might be different. I imagine most of the time the link to the lock file should be a link to the blob (contents of the file) from a specific commit.